docs/sources/writing-guide/references/index.md

```diff
@@ -81,6 +77,28 @@ You can refer to the table below for all file path based relrefs between these f
 | `/docs/sources/branch/other.md`  | `/docs/sources/branch/_index.md` | `{{</* relref "_index.md" */>}}`           |
 | `/docs/sources/branch/other.md`  | `/docs/sources/leaf/index.md`    | `{{</* relref "../leaf/index.md" */>}}`    |
 
+
+## Versions and cross-repository linking
+
+For Hugo's purposes, you can't address other versions of the docs, such as a version-specific archived docs set (`https://grafana.com/docs/grafana/v8.5/` and so forth) or `/next/` docs for links in content residing in `/latest/`, using Hugo references.
```

Comment:
This isn't a Hugo thing, this is a limitation of our local webserver environments.

With the `make-docs` script, it is now possible to build multiple documentation sets together and I have demonstrated cross project link checking in https://github.com/grafana/tempo/pull/2299.

Reply:
```suggestion
For Grafana's webserver environments, you can't address other versions of the docs, such as a version-specific archived docs set (`https://grafana.com/docs/grafana/v8.5/` and so forth) or `/next/` docs for links in content residing in `/latest/`, using Hugo references.
```